src/window/advisor/ratings.c

```diff
@@ -1,6 +1,9 @@
 #include "ratings.h"
 
 #include "city/ratings.h"
+#include "city/data_private.h"
```

Comment:
This applies to all new includes (I'm only commenting here to avoid repetition), but please keep then sorted by alphabetical order.

Also, from a coding standpoint, exposing `data_private.h` to the advisors doesn't seem like a good idea. A best bet would be to create new functions in `culture.c` that return the average values, instead of fetching them directly from the private city data structs.